tests/terraform/checks/resource/aws/test_LaunchConfigurationEBSEncryption.py

```diff
@@ -31,15 +31,14 @@ def test(self):
             "aws_instance.fail4",
             "aws_instance.fail5",
             "aws_instance.fail_empty_root_list",
-            "aws_instance.fail_empty_ebs_list",
             "aws_launch_configuration.fail",
         }
 
         passed_check_resources = {c.resource for c in report.passed_checks}
         failed_check_resources = {c.resource for c in report.failed_checks}
 
         self.assertEqual(summary["passed"], 5)
-        self.assertEqual(summary["failed"], 8)
+        self.assertEqual(summary["failed"], 7)
```

Comment:
can you also add a check for `unknown` here?

Reply:
Unknown is not recognized as part of the test result